modules/mobile-agents/android/client/client/src/main/java/org/wso2/emm/agent/services/Operation.java

```diff
@@ -1016,6 +1019,13 @@ private void installApplication(JSONObject data, org.wso2.emm.agent.beans.Operat
 		}
 	}
 
+	public void disenrollDevice(org.wso2.emm.agent.beans.Operation operation) {
```

Comment:
This method is called remotely and status will send back in the response itself.